ballerina/llm_client_azure_open_ai.bal

```diff
@@ -69,10 +72,18 @@ isolated distinct client class AzureOpenAIModel {
             return error("No completion choices");
         }
 
-        string? resp = choices[0].message?.content;
+        ChatCompletionMessageToolCalls? toolCalls = choices[0].message?.tool_calls;
+
+        if toolCalls is () {
+            return error(NO_RELEVANT_RESPONSE_FROM_THE_LLM);
```

Comment:
Don't we have tests for these?

Reply:
No, Currently the mock tests defined for only when there is tool results
Will add tests for these as a separate PR

Reply:
https://github.com/ballerina-platform/ballerina-library/issues/7903